model/dotnet/runtime-metrics.yaml

```diff
@@ -9,7 +9,7 @@ groups:
       This metric reports the same values as accessing [`Environment.ProcessorCount`](https://learn.microsoft.com/dotnet/api/system.environment.processorcount).
     instrument: updowncounter
     unit: "{cpu}"
-    stability: experimental
+    stability: stable
 
   - id: metric.dotnet.process.cpu.time
     type: metric
```

Comment:
I think we should mark this metric stable too since it's de-facto stable and .NET treats it as such (along with `cpu.mode` attribute.

There are a couple of tricks on how we can do it:
1. add an exception to the policy similarly to https://github.com/open-telemetry/semantic-conventions/blob/3fa52acca3ec1530f9b88a38a29bfd876ec2205d/policies/group_stability.rego#L11-L12
2. don't define corresponding yaml. just define things in markdown similarly to https://github.com/open-telemetry/semantic-conventions/blob/ab1362c44ea6f523398aa4d8b47120580365cc81/docs/dotnet/dotnet-dns-metrics.md?plain=1#L26

I'd prefer p1 since it's limited to one attribute that I hope we'll stabilize soon along with `process` and `system` metrics.

Reply:
I've applied this as suggested. Thanks for the hint.